### PR TITLE
[TW-1552] Create overview for /docs/getting-started/installation

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Adds an overview for the installation section

### CODE REVIEW

File: src/components/LeftNav/LeftNavItems.jsx [hunk 1]
```diff
@@ -148,6 +148,10 @@ export const leftNavItems = [
         subParentSlug: '/docs/getting-started/installation/',
         slug: '/docs/getting-started/installation/installation-and-updates/',
```
Comment (petewitcher-postman) [inline src/components/LeftNav/LeftNavItems.jsx:149]:
```suggestion
        slug: '/docs/getting-started/installation/install-overview/',
```

File: src/pages/docs/getting-started/installation/install-overview.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* [Manage your Postman profile](#manage-your-postman-profile)
+* [Configure a proxy server](#configure-a-proxy-server)
+
+## Manage your Postman installation
+
+You can [install Postman](/docs/getting-started/installation/installation-and-updates) on a number of operating systems including [Windows](/docs/getting-started/installation/installation-and-updates/#install-postman-on-windows), [macOS](/docs/getting-started/installation/installation-and-updates/#install-postman-on-mac), and [Linux](/docs/getting-started/installation/installation-and-updates/#install-postman-on-linux) or use the [Postman web app](/docs/getting-started/installation/installation-and-updates/#use-the-postman-web-app). If you use the Postman web app, for best experience download and install the [Postman Desktop Agent](/docs/getting-started/basics/about-postman-agent/#the-postman-desktop-agent) to take advantage of additional features.
+
+Postman Visual Studio Code extension enables you to test and collaborate on your APIs. You can install it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode).
```
Comment (petewitcher-postman) [inline src/pages/docs/getting-started/installation/install-overview.md:39]:
```suggestion
The Postman Visual Studio Code extension enables you to test and collaborate on your APIs. You can install it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode).
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! I threw in a few suggestions ... aaaand they're committed. 😃

File: src/pages/docs/getting-started/installation/install-overview.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    name: "Blog posts"
+  - type: link
+    name: "Announcing Postman v10: The API Platform for an API-First World"
+    url: "https://blog.postman.com/announcing-postman-v10/"
+  - type: link
+    name: "Introducing the Postman Agent: Send API Requests from Your Browser without Limits"
+    url: "https://blog.postman.com/introducing-the-postman-agent-send-api-requests-from-your-browser-without-limits/"
+---
+
+Postman provides several installation options and enables you to customize your user experience. With the free Postman account, you can securely access your APIs from different machines and collaborate with others on API design. To scale up your API lifecycle, take advantage of paid Postman plans. You can also set up a proxy server as an additional security barrier.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/installation/install-overview.md:25]:
```suggestion
Postman provides several installation options and enables you to customize your user experience. With a free Postman account, you can securely access your APIs from different machines and collaborate with others on API design. To scale up your API lifecycle, take advantage of paid Postman plans. You can also set up a proxy server as an additional security barrier.
```

**Review by mark-anthony-postman — APPROVED** [review]
Looks great--thanks for creating this! 🌟

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**